electron-builder.yml

```diff
@@ -38,7 +38,7 @@ win:
     #     - arm64
   signingHashAlgorithms:
     - sha256
-  sign: "./sign-win.js"
```

Comment:
Took the opportunity to move this script to where the other scripts are instead of repo root since I was testing it.